src/main/java/hudson/plugins/analysis/views/DetailFactory.java

```diff
@@ -199,7 +200,7 @@ else if (link.startsWith("type.")) {
      *            the default encoding to be used when reading and parsing files
      * @return the detail view
      */
-    protected AttributeDetail createAttributeDetail(final AbstractBuild<?, ?> owner, final DefaultAnnotationContainer annotations,
+    protected AttributeDetail createAttributeDetail(final Run<?, ?> owner, final DefaultAnnotationContainer annotations,
```

Comment:
what happens if a subclass has overridden this method using the `AbstractBuild` signature and some other method now calls through with a parameter typed as `Run` (even though it's an AbstractBuild instance)? their override will not be called any more.